gatsby_fp/src/components/fields.js

```diff
@@ -0,0 +1,87 @@
+import React from "react"
+import GenInfoTable from "./fieldComponents/genInfoTable"
+import Education from "./fieldComponents/education"
+import ProfService from "./fieldComponents/profService"
+import PatentInv from "./fieldComponents/patentInv"
+import UnivService from "./fieldComponents/univService"
+import ProfSocieties from "./fieldComponents/profSocieties"
+import IntellCont from "./fieldComponents/intellCont"
+import Section from "./fieldComponents/section"
```

Comment:
Since "section" is all lowercase in this filename, you probably need to rename the file from"Section.js" to "section.js" so it matches. POSIX operating systems such as Mac and Linux are case-sensitive, unlike Windows, which could explain why this is not a problem on your local machine. From what I recall using Windows, you will need to do 2 renames... like first make it "sectionx.js" and then "section.js" (otherwise Windows doesn't think you're doing anything at all, and the rename might fail).